src/components/fold-media.css

```diff
@@ -0,0 +1,7 @@
+.FoldMedia .AttributionEditor-metadata {
+  top: -75px;
+}
```

Comment:
True. I'm not attached to the overlap but want to stay consistent with @d4tocchini 's intention here.